corona-warn-app/cwa-app-android
# Federation/EUR key-package file support (EXPOSUREAPP-3025)

## Description
This adds build time config support for switching the keyfile download behavior from countries to *EUR*.

## How to test
* Update your `test_environments.json` with the latest variant.
* Switch to an environment that supports EUR and observe logcat download behavior.
